lib/elixir/lib/io.ex

```diff
@@ -233,15 +233,49 @@ defmodule IO do
   @doc """
   Inspects and writes the given `item` to the device.
 
+  It's important to note that it returns the given `item` unchanged.
+  This makes it possible to insert an `IO.inspect/2` call into
+  virtually any place of the code or in the middle of a pipeline
```

Comment:
"This makes it possible to "spy" on values by inserting an `IO.inspect/2` call almost anywhere in your code, even in the middle of a pipeline.", might be better phrasing?